Makefile

```diff
@@ -12,6 +12,14 @@ clean:  ## Remove all build artifacts
 test:  ## Run the library test suite
 	tox
 
+requirements: ## install development environment requirements
+	pip install -r requirements/pip.txt
+	pip install -r requirements/pip_tools.txt
+	pip install -r requirements/base.txt
+	pip install -r requirements/test.txt
+	pip install -r requirements/tox.txt
+	pip install -r requirements/ci.txt
+
```

Comment:
Generally we use the `make requirements` rule to install the requirements that a dev would need to work on the project. For this project, I think that's just `pip.txt`, `pip_tools.txt`, and `test.txt`. Note that `test.txt` is a superset of `base.txt`.

Reply:
Requirements have been edited.